code/src/java/plugin/doomsdaybook/RandomNamePlugin.java

```diff
@@ -22,15 +22,12 @@
 import gmgen.GMGenSystemView;
 import gmgen.pluginmgr.messages.AddMenuItemToGMGenToolsMenuMessage;
 import gmgen.pluginmgr.messages.RequestAddTabToGMGenMessage;
-
 import java.awt.Component;
 import java.awt.event.ActionEvent;
 import java.io.File;
-
```

Comment:
This is not consistent with the rest of our files, and is not consistent with the eclipse formatting we use... so these deleted spaces will be reverted by eclipse users.